overrides/ios-override.json

```diff
@@ -40,7 +40,18 @@
             "state": "enabled"
         },
         "autofill": {
-            "state": "enabled"
+            "state": "internal",
```

Comment:
If my current understanding of how the existing autofill flag is used, this will be a problem. I think setting the top level flag to internal will break email for existing users

Reply:
The reason I did this was that I couldn't see anywhere that's using the Privacy Config's `.autofill` key. We only seem to be using the `FeatureFlagger`.

Reply:
@graeme the trouble here is the content scope scripts. The config in its entirety is passed to the content scope scripts which execute JavaScript in every tab. These scripts are using the `autofill` key to control the autofill feature on the webpages.

Reply:
Oh right yes, of course. I was told about this.